setup.py

```diff
@@ -17,8 +17,8 @@
     url="https://github.com/great-expectations/airflow-provider-great-expectations",
     install_requires=[
         "apache-airflow>=2.1",
-        "great-expectations>=0.13.14",
-        "sqlalchemy>=1.3.16",
+        "great-expectations>=0.15.34",
+        "sqlalchemy>=1.4.44",
```

Comment:
Thanks for adding this. I think we should probably pin with an `==` for good measure.

Reply:
Hi @denimalpaca , sorry that I just saw your comment. Updated the install_requires with ==. Btw, do we have auto-squash commits setup in our PR? Or I need to do it on my local? Thank you again ~

Reply:
@zhangchi1 You need to rebase on main now, we just changed `setup.py` to `setup.cfg`. Auto-squash commits are enabled. I'm just doing a little local test on the req updates then I'll approve and @kaxil or I can squash and merge after you rebase. Thank you for contributing!

Reply:
sorry about suggesting the `==` pinning, didn't realize it would potentially cause problems!